### ISSUE TITLE
Edge case: C++ Templated operator overload member function

### ISSUE THREAD

**Issue by JohannesMP** [20Tauri/DoxyDoxygen#36]
With these settings: https://gist.github.com/JohannesMP/bfbd47ffe4879d3ea691

This code:

```
template <class T>
Foo<T>& Foo<T>::operator=(T const &bar )
{
  //...
}
```

fails to generate a comment body:

<img src=https://i.imgur.com/ubYB1RX.png />

**Comment by 20Tauri** [comment]
Thank for reporting,
Problem was occurred with all operators prefixed with a scope.
I have fixed it in 0.33.1 (released in evolution branch)

**Comment by JohannesMP** [comment]
Thanks for the blazing fast reply!
